includes/discovery/sensors/count/cisco.inc.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/*
+ * LibreNMS - Cisco count sensors
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ *
+ * @package    LibreNMS
+ * @subpackage webui
+ * @link       http://librenms.org
+ * @copyright  2021 LibreNMS
+ * @author     LibreNMS Contributors
+*/
+
+$tables = [
+    ['num_oid' => '.1.3.6.1.4.1.9.9.661.1.3.1.1.1.',    'oid' => 'c3gImsi',                   'state_name' => 'c3gImsi',                    'mib' => 'CISCO-WAN-3G-MIB', 'descr' => 'IMSI'],
+    ['num_oid' => '.1.3.6.1.4.1.9.9.661.1.3.1.1.2.',    'oid' => 'c3gImei',                   'state_name' => 'c3gImei',                    'mib' => 'CISCO-WAN-3G-MIB', 'descr' => 'IMEI'],
+    ['num_oid' => '.1.3.6.1.4.1.9.9.661.1.3.2.1.6.',    'oid' => 'c3gGsmNumberOfNearbyCell',  'state_name' => 'c3gGsmNumberOfNearbyCell',   'mib' => 'CISCO-WAN-3G-MIB', 'descr' => 'Nearby cells'],
+];
+
+foreach ($tables as $tablevalue) {
+    $temp = snmpwalk_cache_multi_oid($device, $tablevalue['oid'], [], $tablevalue['mib']);
+    $cur_oid = $tablevalue['num_oid'];
+    $state_name = $tablevalue['state_name'];
+    foreach ($temp as $index => $entry) {
+        //Discover Sensors
+        $descr = ucwords($temp[$index][$tablevalue['descr']]);
+        if ($state_name == 'ffgfggf' || $state_name == '345tyh' || $state_name == 'dgdfgdd' || $state_name == 'gfgfgf') {
```

Comment:
This code here seems interesting :) you probably forgot to cleanup :) 
State_name doesn't mean anything here anyway, this is not a state sensor. You should completely get rid of it.

Reply:
Thanks, indeed needed a cleanup. fixed it locally, will be fixed in next push.